test/functional/tests/io/trim/test_trim.py

```diff
@@ -131,36 +132,54 @@ def test_trim_propagation():
     with TestRun.step(f"Disable udev"):
         Udev.disable()
 
-    with TestRun.step(f"Prepare cache instance in WB with one core"):
+    with TestRun.step("Start cache in Write-Back mode and add core device"):
         cache = casadm.start_cache(cache_dev, CacheMode.WB, force=True)
         core = cache.add_core(core_dev)
-        cache.set_cleaning_policy(CleaningPolicy.nop)
+
+    with TestRun.step("Disable cleaning policy and sequential cutoff"):
         cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+
+    with TestRun.step("Purge cache and reset cache counters"):
         cache.purge_cache()
+        cache.reset_counters()
 
-    with TestRun.step(f"Fill exported object with dirty data"):
+    with TestRun.step(f"Prepare I/O to fill exported object with dirty data"):
```

Comment:
unnecessary f-string
how is I/O prepared here? (preparing config for I/O tool is not a real test step)